src/lib/core/common-behaviors/common-module.ts

```diff
@@ -104,6 +112,18 @@ export class MatCommonModule {
     this._document.body.removeChild(testElement);
   }
 
+  /** Checks whether the material version matches the cdk version */
+  private _checkCdkVersionMatch(): void {
+    if (!CDK_VERSION || VERSION.full !== CDK_VERSION.full) {
+      console.warn(
+          'The Angular Material version ' + VERSION.full + ' does not match ' +
+          'the Angular CDK version ' + (CDK_VERSION ? CDK_VERSION.full : '') + '.\n' +
+          'Please install Angular CDK ' + VERSION.full + ': ' +
+          'npm install @angular/cdk@' + VERSION.full
```

Comment:
I would tweak the message to be:
```
The Angular Material version (xxx) does not match the Angular CDK version (yyy). Please
ensure the versions of these two packages exactly match.
```

Since people might want to keep their cdk version and change material. Also people might be using `yarn`, so I would avoid giving an actual command.

Reply:
I updated the message accordingly